core/api.go

```diff
@@ -132,6 +133,12 @@ func (a *api) Start() {
 			}
 		}
 
+		messages := v0.Group("/messages")
```

Comment:
this feels a bit weird having the two separate paths like this? not sure how else it should be done... is this pretty standard practice?

Reply:
yeah, agreed. same for files. my thought there was it's nice to be able to paginate across all files / messages... but maybe not needed. can revisit.